### ISSUE TITLE
[PL][ENG][Other] Transmog Bow/Gun/Crossbow

### ISSUE THREAD

**Issue by Raquieta** [Atlantiss/BugTracker#4581]
[PL]
Jak wiemy powinna być możliwość zmieniania wyglądu broni odległościowych na wszystkie typy bez znaczenia czy to łuk, kusza czy pistolet.
Tak więc w skrócie:
Gun > Crossbow - działa
Crossbow > Gun - działa
Problem natomiast pojawia się gdy chcemy zmienić wygląd kuszy, lub pistoletu na łuk.
Każda próba spotyka się z dokładnie takim samym efektem jak kiedy próbujemy zmienić wygląd broni na inną, która nie ma żadnych atrybutów (stamina etc.), poprostu nic się nie dzieje.

[ENG]
As we know, there should be a possibility to change the look of ranged weapons between all the types, no matter if it's bow, crossbow or gun.
In short:
Gun > Crossbow - works
Crossbow > Gun - works
Problem appears when we want to change the look of crossbow or gun to look like bow.
Each try ends up with exactly same result as when we'd try to change some weapons look into another that has no attributes (stamina etc.), just nothing happens.

**Comment by Sthibodeaux** [comment]
this was introduced in mists of pandaria, transmog was item specific in cata, meaning guns have to be guns, bows have to be bows, 1hswords have to be 1hswords, etc.

**Comment by Raquieta** [comment]
You are wrong.
These are Transmogrification rules for 4.3.4 accounced by Blizzard when 4.3 patch launched.
http://us.battle.net/wow/en/forum/topic/3196342583
Ranged weapons are exception since day 1.
Find the line saying:
• Guns, Crossbows, and Bows can be used to transmogrify Guns, Crossbows, or Bows.

Also, since I remember, on all private cata servers you were able to do it, and it also worked here before. As I said in original post, transmogrifying gun to crossbow and vice versa WORKS here, but not when you're trying to use a regular BOW as the base, or to use it's model, and that's the bug.

Mist of Pandaria last patch only introduced the transmog between the meele weapon types, because people were doing it anyways using the macro since 4.3 patch, which was never prohibited.

**Comment by Wolffenstein** [comment]
Fixed in rev 4701.
